apps/full-site-editing/full-site-editing-plugin/site-editor/index.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Site Editor experiment file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Enables/Disables site editor experiment per blog sticker.
+ */
+function initialize_site_editor() {
+	if ( ! is_site_editor_active() ) {
+		return;
+	}
+
+	// Force enable required Gutenberg experiments if they are not already active.
+	add_filter(
+		'option_gutenberg-experiments',
+		function( $experiments_option ) {
+			if ( empty( $experiments_option['gutenberg-full-site-editing'] ) ) {
+				$experiments_option['gutenberg-full-site-editing'] = 1;
+			}
+
+			if ( empty( $experiments_option['gutenberg-full-site-editing-demo'] ) ) {
+				$experiments_option['gutenberg-full-site-editing-demo'] = 1;
```

Comment:
When I turn off the site editor experiment via Gutenberg settings, I get the following error:

```
[04-Mar-2020 21:41:14 UTC] Warning: Illegal string offset 'gutenberg-full-site-editing-demo' in /home/wpcom/public_html/wp-content/plugins/full-site-editing-plugin/dev/site-editor/index.php on line 27 [noahtallentest11.wordpress.com/wp-admin/admin-ajax.php] [wp-includes/class-wp-hook.php:289 A8C\FSE\{closure}(), wp-includes/plugin.php:206 apply_filters(''), wp-includes/option.php:113 apply_filters('option_gutenberg-experiments'), /plugins/gutenberg-core/7.6.0-rc.1/lib/load.php:22 get_option(), /plugins/gutenberg-core/7.6.0-rc.1/lib/template-parts.php:12 gutenberg_is_experiment_enabled(), wp-includes/class-wp-hook.php:287 gutenberg_register_template_part_post_type(), wp-includes/class-wp-hook.php:311 apply_filters(''), wp-includes/plugin.php:478 do_action(Array), wp-settings.php:515 do_action('init'), wp-config.php:7 require('wp-settings.php'), wp-load.php:37 require_once('wp-config.php'), wp-admin/admin-ajax.php:22 require_once('wp-load.php')]
```

Reply:
This should also be fixed as part of bc622aaeb7a22eaf2c066649a01dfcd127652ff1.